app/helpers/games_helper.rb

```diff
@@ -14,4 +14,28 @@ def black_tile?(x,y)
   def is_square_for_piece_to_be_moved?(the_piece, piece)
     return the_piece && the_piece == piece
   end
+
+  def white_king
+    return @game.pieces.where(piece_number: 4).first
+  end
+  
+  def white_queenside_rook
+      return @game.pieces.where(x_position: 0, y_position: 0).first
```

Comment:
We should probably specify the piece_number here as well. What if a piece that isn't a rook is at [0, 0]?

Reply:
Any other piece there would of had to move to get there so should still fail the tests.